### ISSUE TITLE
Suggestion: ReAdd im@s pics on descript pages

### ISSUE THREAD

**Issue by kaylobadboy** [ewhal/nyaa#110]
The idol png pics previously on nyaa were archived. Here's a mega link with the 14 images that used to display on the info pages

https://mega.nz/#F!mwETBaQS!N5E8e9JwRCi0sW-CviFdlA

**Comment by S0X-LT** [comment]
Replace the im@s idols with the ones from Love Live.

Actually, make it so you can put whatever, so if anyone deploys their own version of the site, they can replace them with whatever they want.

Or not implement then at all, to reduce bloat.

**Comment by Argolics** [comment]
I don't think this were a good idea at all for original nyaa, so we probably should move on and at most make tpb style up/downvote things for people to see how much this particular release is received.
I doubt any kind of favorites system would be useful for torrent tracker, and all those rating things are prone to cheating and abusing, all for very little benefit.

**Comment by gensuihime** [comment]
This is probably something left for after the site's front/backend is stabilized? Also, this is more of a non-critical feature that would probably slow down website load times. Doesn't seem like it holds a lot of importance at the moment.
